### PR TITLE
Rerelease: HipChat

### PR DESCRIPTION
**Pull request by epaulet**
addresses issue #67

### CODE REVIEW

File: lib/integrations/hip_chat.rb [hunk 1]
```diff
@@ -1,83 +1,68 @@
...
-    def run_completion_message
-      "is complete!"
-    end
-
-    def run_error_message
-      "has encountered an error!"
-    end
+class Integrations::HipChat < Integrations::Base
+  def self.key
+    "hip_chat"
```
Comment (shosti) [inline lib/integrations/hip_chat.rb]: Why `hip_chat` and not `hipchat`?

**Comment by shosti** [comment]
looks good other than the weird key.

File: lib/integrations/hip_chat.rb (hunk 1 above)
Comment (epaulet) [inline lib/integrations/hip_chat.rb]: `hipchat` to camelcase is `Hipchat` which is actually incorrect. The name is `HipChat`.

Comment (shosti) [inline lib/integrations/hip_chat.rb]: Oh, we automatically camelize? That's kind of annoying, it's always referred to as `hipchat` in lower case (e.g. in the gem and whatnot). But if we do it automatically I guess it's OK.

Comment (epaulet) [inline lib/integrations/hip_chat.rb]: Yeah, it's just horribly inconsistent :-1:

**Pull request merged**